joomla/joomla-cms
# Consistency Title in Categories

This change applies to all components that use com_categories except for User Notes.
It adds more consistency to the Title when adding a new category or when editing a category.

Currently when a category is added or when a category is edited, the title puls the strings from en-GB.com_categories.ini. Except when a categories in Articles is edited, it shows a string that is currently in en-GB.com_content.ini though the constant of that one is COM_CATEGORIES_CATEGORY_EDIT_TITLE which indicates it should be in en-GB.com_categories.ini

In the latter file this constant already exists and shows the Titles on the respective views:
Category Manager: Add A New %s Category
Category Manager: Edit A New %s Category
Where %s is the components name (e.g Category Manager: Add A New Banners Category
Category Manager: Edit A New Banners Category)
The use of Manager is however deprecated, and should be changed.
This patch changes the strings into:
%s: New Category and %s: Edit Category where %s is the components name.

To test:
View the Titles in the pages where categories can be added and edited before applying the patch:
Articles -> Categories: 
Click New - The title shows "Category Manager: Add A New Articles Category"
Edit a category by either clicking on the title or check the check box and click on Edit in the toolbar - The title shows: "Article: Edit Category"

Banners -> Categories:
Click New - Title shows "Category Manager: Add A New Banners Category"
Edit a category by either clicking on the title or check the check box and click on Edit in the toolbar - The title shows: "Category Manager: Edit A New Banners Category"

This applies also to the Category views in Contacts, News Feeds, and Web Links with the comment that Banners is replaced by Contacts, News Feeds, and Web Links in the respective views of those components.

After applying the patch each of the above view shows the title:
Articles: Edit Category *
Articles: New Category *
(*) Articles is replaced by Banners, Contacts, News Feeds, and Web Links in the respective views of those components.

For consistency I also changed COM_CATEGORIES_CATEGORIES_BASE_TITLE="Category Manager" into COM_CATEGORIES_CATEGORIES_BASE_TITLE="Categories"
